DSpace/DSpace
# support CSV import to schema "person"

## References
* Fixes https://github.com/DSpace/DSpace/issues/3006

## Description
This PR adds a fix to the DSpaceCSV.java class to make it possible to better distinguish a normal "person" metadata field from an authority field.
Any field that doesn't start with [AUTHORITY_TYPE] is now treated as a regular field without the chance of it being detected as an authorityType, we made sure that this only happens if the String is surrounded by brackets. 

## Instructions for Reviewers

List of changes in this PR:
* Made sure that the AuthorityType evaluating is only done for Strings between brackets
* Added test to ensure that the CSV import with person metadata works now

How to test this PR:
* Make a CSV with a person.birthDate metadata field in it with a value
* Import the CSV and verify that the person.birthDate is added to the imported Item

## Checklist
